woocommerce/woocommerce
# Product Collection: Fix Preview badge border-radius and alignment

### Changes proposed in this Pull Request:





The Preview badge's bottom left and right corners now have border-radius set to 0px to ensure they smoothly morph into the block's stroke. The badge is visually aligned with the furthest edge of the block's right border.

Closes #47646 



### How to test the changes in this Pull Request:



1. Go to "Products by Attribute" template
2. Add a Product Collection block if not already present
3. Select the Product Collection block
4. Verify that the Preview badge's bottom left and right corners have border-radius set to 0px. See screenshot below:
![image](https://github.com/woocommerce/woocommerce/assets/16707866/e9cbe89e-b8d5-45e3-88bc-3ddd65ba13ea)




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Product Collection: Fix the Preview badge's corner radius

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
